src/components/ImageScroller.js

```diff
@@ -88,7 +88,16 @@ export default function ImageScroller(props) {
   const classes = useStyle();
   const [maxImages, setMaxImages] = useState(MAX_IMAGES_INCREMENT);
   const imageScrollerRef = useRef(null);
-  let [rotation, setRotation] = useState(imageRotation);
+  let objRotation = {};
+  if (selectedImage) {
+    const selectedIndex = images.indexOf(selectedImage);
+    objRotation[selectedIndex] = imageRotation;
+  } else {
+    images.map((_, idx) => {
+      objRotation[idx] = imageRotation;
```

Comment:
It doesn't really make sense to apply image rotation to all images if none is selected.
Just leave all other images unrotated and apply the rotation to the selected image only.